src/map/script.cpp

```diff
@@ -23696,6 +23696,30 @@ BUILDIN_FUNC(round) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(getequiptradability) {
+	int i = -1, num;
+	TBL_PC *sd;
+
+	num = script_getnum(st, 2);
+
+	if (!script_charid2sd(3, sd)) {
+		script_pushint(st, 0);
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	if (equip_index_check(num))
+		i = pc_checkequip(sd, equip_bitmask[num]);
+	if (i >= 0) {
+		bool tradable = (sd->inventory.u.items_inventory[i].expire_time == 0 &&
+			(!sd->inventory.u.items_inventory[i].bound || pc_can_give_bounded_items(sd)));
+		script_pushint(st, tradable);
+	}
+	else
+		script_pushint(st, 1);
```

Comment:
Shouldn't this be returning false instead of 1(true)? The index value would be bad (returned -1) so this should fail as well.

Reply:
Yeah I agree with aleos. This might not be a script command failure case, but it should return not tradable at least.